public/css/style.css

```diff
@@ -70,6 +70,11 @@ textarea {
   font-size: 14px;
 }
 
+textarea {
+  resize: vertical;
+  overflow: auto;
```

Comment:
I don't see that this is required either? `textarea` already has `overflow: auto` as a default in any case. 
```suggestion
```